FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/PipelineRegistrationExtensions.cs

```diff
@@ -64,5 +65,25 @@ public static IServiceCollection AddSchemaConverters(this IServiceCollection ser
 
             return services;
         }
+
+        public static IServiceCollection AddFhirSpecificationProviders(this IServiceCollection services)
+        {
+            services.AddSingleton<IFhirSpecificationProvider, R4FhirSpecificationProvider>();
+
+            services.AddTransient<R4FhirSpecificationProvider>();
```

Comment:
What's the consideration to mark the specification provider as `Transient`? Seems it won't change at runtime.

Reply:
Fixed, thanks.